README.md

```
@@ -1,51 +1,86 @@
 # Predator
 
 <p align="center">
-<img src="asset/Predatorv1.png" width="200">
+<img src="logov4.png" width="200">
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)